<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Git::Internal::GitSubmitPanel</class>
 <widget class="QWidget" name="Git::Internal::GitSubmitPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>201</width>
    <height>189</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="infoGroup">
     <property name="title">
      <string>General Information</string>
     </property>
     <layout class="QFormLayout" name="infoFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="repositoryLabelLabel">
        <property name="text">
         <string>Repository:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="repositoryLabel">
        <property name="text">
         <string>repository</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="branchLabelLabel">
        <property name="text">
         <string>Branch:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="branchLabel">
        <property name="text">
         <string>branch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="editGroup">
     <property name="title">
      <string>Commit Information</string>
     </property>
     <layout class="QFormLayout" name="editFormLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="authorLabel">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="authorLineEdit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="emailLabel">
        <property name="text">
         <string>Email:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="emailLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
